app/_data/docs_nav_gateway_3.0.x.yml

```
@@ -11,504 +11,529 @@ items:
-          - text: Configure Key Authentication
-            url: /get-started/comprehensive/secure-services
-          - text: Configure Load Balancing
-            url: /get-started/comprehensive/load-balancing
-          - text: Workspaces and Teams
-            url: /get-started/comprehensive/manage-teams
-          - text: Publish and Consume Services
-            url: /get-started/comprehensive/dev-portal
-
-  - title: Plan and Deploy
-    icon: /assets/images/icons/documentation/icn-deployment-color.svg
-    items:
-      - text: Running Kong as a Non-Root User
-        url: /plan-and-deploy/kong-user
-      - text: Resource Sizing Guidelines
-        url: /plan-and-deploy/sizing-guidelines
-      - text: Hybrid Mode
-        url: /plan-and-deploy/hybrid-mode
+          - text: Start Kong Gateway Securely
+            url: /kong-production/running-kong/start-kong-securely
+          - text: Running Kong as a non-root user
+            url: /kong-production/running-kong/kong-user
+          - text: Configure Kong Gateway and SystemD
+            url: /kong-production/running-kong/systemd
+      - text: Kubernetes
         items:
-          - text: Deploy Kong Gateway in Hybrid Mode
-            url: /plan-and-deploy/hybrid-mode/hybrid-mode-setup
-      - text: Kubernetes Deployment Options
-        url: /plan-and-deploy/kubernetes-deployment-options
-      - text: Control Kong Gateway through systemd
-        url: /plan-and-deploy/systemd
-      - text: Performance Testing Framework
-        url: /plan-and-deploy/performance-testing-framework
-      - text: DNS Considerations
-        url: /plan-and-deploy/dns-considerations
-      - text: Default Ports
-        url: /plan-and-deploy/default-ports
+          - text: Kubernetes Deployment Options
+            url: /kong-production/kubernetes/kubernetes-deployment-options
       - text: Licenses
-        url: /plan-and-deploy/licenses
         items:
-          - text: Access Your License
-            url: /plan-and-deploy/licenses/access-license
-          - text: Deploy Your License
-            url: /plan-and-deploy/licenses/deploy-license
-          - text: Monitor License Usage
-            url: /plan-and-deploy/licenses/report
-      - text: Security
+          - text: Overview
+            url: /kong-production/licenses/
+          - text: Access your license
+            url: /kong-production/licenses/access-license
+          - text: Deploy Enterprise License
+            url: /kong-production/licenses/deploy-license
+          - text: Monitor Licenses usuage
+            url: /kong-production/licenses/report
+      - text: Networking
+        items:
+          - text: Default Ports
+            url: /kong-production/networking/default-ports
+          - text: DNS considerations
+            url: /kong-production/networking/dns-considerations
+          - text: Network and Firewall
+            url: /kong-production/networking/network
+      - text: Kong.conf Configuration File
```

```suggestion
      - text: Kong Configuration File
```
Could just be a personal preference and I know we have "kong.conf Configuration File" in the Google Sheet. Seeing "kong.conf" was just kind of jarring in the sidebar nav.